README.md

```diff
@@ -317,86 +324,128 @@ import { define, vdom } from 'skatejs';
 
 
 
-### `define(name, definition)`
+### Using the Platform
 
-The `name` is a string that is the tag name of the custom element that you are creating. It must be a "valid custom element name" as specified [in the spec](http://w3c.github.io/webcomponents/spec/custom/#custom-elements-core-concepts). For example, `my-component`, is a valid custom element name.
+Skate focuses on staying as close to the platform as possible. This means that instead of having to use `skate.define()` you can just use `customElements.define()`, which is built-in. You can still use `skate.define()`, but it's opt-in, and only serves to enhance what the browser already gives you.
 
-The `definition` argument is a class / constructor or object literal of component definition to use for your custom element. The recommended way is to use ES2015 classes:
+The lifecycle methods in Skate also use the `*Callback` naming convention. For example, the `render` lifecycle is represented by the `renderCallback()` on the custom element `prototype`. Things like `props` follow the convention of `observedAttributes` and are provided as static getters.
 
-```js
-import { Component, define } from 'skatejs';
+If you're using IE9 and 10, transpiling class extension is limited to instance methods and properties. Statics don't get added to the chain. However, Skate supports calling `extend()` on a component which you want to extend. This means that in older browsers, you can do something like:
 
-const MyComponent = define('my-component', class extends Component {});
+```js
+const MyComponent1 = skate.Component.extend();
+const MyComponent2 = MyComponent1.extend();
 ```
 
-A simpler way, especially for ES5 users, would be to just pass an object literal.
+Recently, we've deprecated several old methods in favour of aligning closer to the native APIs. These methods still work but will be removed in a future version. Not all methods / properties are listed here, only the ones that have been deprecated and what they're superceded by.
```

Comment:
Thought I fixed that, but I think that was a search and replace of the misspelling. It should be supersede.